include/zephyr/bluetooth/mesh/access.h

```diff
@@ -41,14 +41,21 @@
 extern "C" {
 #endif
 
-#define BT_MESH_ADDR_UNASSIGNED    0x0000
-#define BT_MESH_ADDR_ALL_NODES     0xffff
-#define BT_MESH_ADDR_RELAYS        0xfffe
-#define BT_MESH_ADDR_FRIENDS       0xfffd
-#define BT_MESH_ADDR_PROXIES       0xfffc
-#define BT_MESH_ADDR_DFW_NODES     0xfffb
-#define BT_MESH_ADDR_IP_NODES      0xfffa
-#define BT_MESH_ADDR_IP_BR_ROUTERS 0xfff9
+/**
+ * @name Group addresses
+ * @{
+ */
+#define BT_MESH_ADDR_UNASSIGNED    0x0000   /**< unassigned */
+#define BT_MESH_ADDR_ALL_NODES     0xffff   /**< all-nodes */
+#define BT_MESH_ADDR_RELAYS        0xfffe   /**< all-relays */
+#define BT_MESH_ADDR_FRIENDS       0xfffd   /**< all-friends */
+#define BT_MESH_ADDR_PROXIES       0xfffc   /**< all-proxies */
+#define BT_MESH_ADDR_DFW_NODES     0xfffb   /**< all-directed-forwarding-nodes */
+#define BT_MESH_ADDR_IP_NODES      0xfffa   /**< all-ipt-nodes */
+#define BT_MESH_ADDR_IP_BR_ROUTERS 0xfff9   /**< all-ipt-border-routers */
+/**
+ * @}
+ */
 
 #define BT_MESH_KEY_UNUSED        0xffff
```

Comment:
Could you please also add description for macros at lines 60 - 73?

Reply:
I couldn't come up with any better than this:
```
/**
 * @name Predefined key indexes
 * @{
 */
#define BT_MESH_KEY_UNUSED        0xffff          /**< Key unused */
#define BT_MESH_KEY_ANY           0xffff          /**< Any key index */
#define BT_MESH_KEY_DEV           0xfffe          /**< Device key */
#define BT_MESH_KEY_DEV_LOCAL     BT_MESH_KEY_DEV /**< Local device key */
#define BT_MESH_KEY_DEV_REMOTE    0xfffd          /**< Remote device key */
#define BT_MESH_KEY_DEV_ANY       0xfffc          /**< Any device key */
/**
 * @}
 */
```

Reply:
added!